packages/scripts/src/ManualRelay.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+// This script allows you to manually relay a particular across deposit.
+// Example:
+// NODE_URL_1=https://mainnet.infura.io/v3/SOME_PROJECT_ID NODE_URL_10=https://optimism-mainnet.infura.io/v3/SOME_PROJECT_ID \
+//   ./src/ManualRelay.js --chainId 10 --depositId 100 --network mainnet_mnemonic
+// Note: the NODE_URL_<chainId> environment variable must be provided for the chain id of the deposit and mainnet so
+// the script can retrieve information from both networks.
+// Optional env overrides:
+//   BRIDGE_ADMIN_ADDRESS
+//   BRIDGE_DEPOSIT_ADDRESS
+//   L2_END_BLOCK_NUMBER
+//
+// Note: if you want to run with GCKMS keys, the command will look like this:
+// NODE_URL_1=https://mainnet.infura.io/v3/SOME_PROJECT_ID NODE_URL_10=https://optimism-mainnet.infura.io/v3/SOME_PROJECT_ID \
+//   GOOGLE_APPLICATION_CREDENTIALS=/your/credentials/file.json ./src/ManualRelay.js --chainId 10 --depositId 100 \
+//   --network mainnet_gckms --keys your_gckms_key_name
+
+const argv = require("minimist")(process.argv.slice(), { string: ["depositId", "chainId"] });
+
+const { getWeb3ByChainId } = require("@uma/common");
+const { InsuredBridgeL1Client, InsuredBridgeL2Client, Logger } = require("@uma/financial-templates-lib");
+const sdk = require("@uma/sdk");
+const { getAddress } = require("@uma/contracts-node");
+
+async function main() {
+  if (!argv.chainId) {
```

Comment:
worth while validating that this `chainId` is part of the known list of: 10, 42161 or 288?

Reply:
Any reason we don't want to allow people to add addresses via env and use this on a testnet deployment?

Reply:
that's fair. main issue I had was you could provide an invalid chainId then it would fail in a confusing way but I guess leaving it less opinionated to open up test net usage is better.

Reply:
I added a generic error message to the end of the script suggesting that the user double check their chain id.

Reply:
I added a specific error if they don't provide a custom deposit box address _and_ their chainId isn't listed in the object.